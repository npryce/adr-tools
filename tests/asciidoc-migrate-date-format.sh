adr init -t adoc
(
	ADR_DATE=12/01/1992
	adr new With Old Date Format
)
adr new With Current Date Format
grep Date: doc/adr/*
adr upgrade-repository
grep Date: doc/adr/*
