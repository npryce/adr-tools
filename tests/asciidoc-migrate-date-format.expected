adr init -t adoc
doc/adr/0001-record-architecture-decisions.adoc
(
	ADR_DATE=12/01/1992
	adr new With Old Date Format
)
doc/adr/0002-with-old-date-format.adoc
adr new With Current Date Format
doc/adr/0003-with-current-date-format.adoc
grep Date: doc/adr/*
doc/adr/0001-record-architecture-decisions.adoc:Date: 1992-01-12
doc/adr/0002-with-old-date-format.adoc:Date: 12/01/1992
doc/adr/0003-with-current-date-format.adoc:Date: 1992-01-12
adr upgrade-repository
grep Date: doc/adr/*
doc/adr/0001-record-architecture-decisions.adoc:Date: 1992-01-12
doc/adr/0002-with-old-date-format.adoc:Date: 1992-01-12
doc/adr/0003-with-current-date-format.adoc:Date: 1992-01-12
